src/env.cpp

```diff
@@ -336,7 +336,7 @@ void env_init(const struct config_paths_t *paths, bool do_uvars, bool default_pa
 
     // Set $USER, $HOME and $EUID
     // This involves going to passwd and stuff.
-    vars.set_one(L"EUID", ENV_GLOBAL, to_string(static_cast<unsigned long long>(geteuid())));
+    vars.set_one(L"EUID", ENV_GLOBAL, std::to_wstring(static_cast<unsigned long long>(geteuid())));
```

Comment:
As a general rule, please don't add unrelated cleanup to a PR.

This has nothing to do with the matter at hand and complicates review.

(of course if it's a line or two in the places you are *already* touching that's fine, but that's not the case here)